var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -44,7 +45,7 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
     variant("hiop", default=False, description="Enable/Disable HiOp")
     variant("ipopt", default=False, description="Enable/Disable IPOPT")
```

Comment:
```suggestion
    variant("ipopt", default=False, description="Enable/Disable IPOPT")
    conflicts("+ipopt ^ipopt+mumps", msg="ExaGO with Ipopt+mumps results in numerical instability")
```
I am not 100% certain on the syntax here, but we should reflect issues seen when building Ipopt with mumps
@pelesh maybe you have a better msg to put here.

Reply:
Perhaps the following is better?
```diff
-    depends_on("ipopt", when="+ipopt")
+    depends_on("ipopt~mumps", when="+ipopt")
``

Reply:
Yes - that would be best

Reply:
updated